shrinker-test/proguard.pro

```diff
@@ -15,3 +15,6 @@
 -keepclassmembernames class com.example.NoSerializedNameMain$TestClassWithoutDefaultConstructor {
   <fields>;
 }
+#-keep class com.example.ClassWithSerializedName {
+#  <init>(...);
+#}
```

Comment:
Is this needed?

Reply:
Good catch, no removed the rules in comments. Also removed the rule above, as that was also not needed. I probably added it while testing different rules.